### ISSUE TITLE
ability to hide autolinks section in commit details

### ISSUE THREAD

**Issue by d13** [gitkraken/vscode-gitlens#2366]
### Discussed in https://github.com/gitkraken/vscode-gitlens/discussions/2365

<div type='discussions-op-text'>

<sup>Originally posted by **JakubFranek** November 22, 2022</sup>
Hello

I would like to know if it is possible to permanently remove the Autolinks menu from the commit detail view as I do not use this feature and it takes up space.

I am talking about the central part in the following screenshot:
![obrazek](https://user-images.githubusercontent.com/33395319/203384049-6850c8e6-96b0-4298-9f94-19d540f22ca4.png)

I tried disabling all the gitlens.autolinks checkboxes I could find in the settings and restart VSCode but it did not help. Did not manage to Google the answer either.

Thanks!</div>

**Comment by Peter-Schorn** [comment]
This annoys me as well. Please add the ability to remove the autolinks section.

**Comment by chivesssss** [comment]
Any news ?

**Comment by thomas-darling** [comment]
Yeah, that section is super annoying - please add an option to hide it.
And while you're at it, please do the same for the Explain section 🙂

**Comment by redtogin** [comment]
I would like to just join in that I think the screen real estate in the Commit Graph Details section is really not efficient. At least for me, half of the content there is not needed. Which means that the change list gets only half of the otherwise available space. 

It would be nice for me personally if the Autolinks, Explain (AI) sections could be disabled, along with the user avatar, user name, and the commit message font size could be decreased.

**Comment by irvnriir** [comment]
"Details View"/"Commit Details view"/"Inspect"/"GitLens Inspect" is the main location which all buttons to review a commit link to . And this situation could be improved by just reordering the Autolinks and Changes views . The only reason i switch from [git graph](https://github.com/mhutchie/vscode-git-graph) is because its not maintained, not fixing this issue in 1.5 years by now is just shame .

**Comment by irvnriir** [comment]
Though yes, the whole menu needs an update

Clear
![image](https://github.com/gitkraken/vscode-gitlens/assets/13949704/135f7871-8122-45aa-9659-1044ea077879)

Current
![1](https://github.com/gitkraken/vscode-gitlens/assets/13949704/2e3dede3-1c16-4093-8b90-1bb3eac5781e)

**Comment by ProudRabbit** [comment]
Any news ? This requirement has not yet been implemented in version 15.3.1.
